core/src/main/scala/kafka/server/AdminManager.scala

```diff
@@ -513,15 +513,21 @@ class AdminManager(val config: KafkaConfig,
     resource -> ApiError.NONE
   }
 
-  private def alterBrokerConfigs(resource: ConfigResource, validateOnly: Boolean,
-                                 configProps: Properties, configEntriesMap: Map[String, String]): (ConfigResource, ApiError) = {
+  private def alterBrokerConfigs(resource: ConfigResource,
+                                 validateOnly: Boolean,
+                                 configProps: Properties,
+                                 configEntriesMap: Map[String, String]): (ConfigResource, ApiError) = {
     val brokerId = getBrokerId(resource)
     val perBrokerConfig = brokerId.nonEmpty
     this.config.dynamicConfig.validate(configProps, perBrokerConfig)
     validateConfigPolicy(resource, configEntriesMap)
     if (!validateOnly) {
-      if (perBrokerConfig)
+      if (perBrokerConfig) {
+        val previousConfigProps = config.dynamicConfig.currentDynamicBrokerConfigs
         this.config.dynamicConfig.reloadUpdatedFilesWithoutConfigChange(configProps)
+        this.config.dynamicConfig.maybeAugmentSSLStorePaths(configProps, previousConfigProps)
```

Comment:
Can you explain why this change is needed?

Reply:
The rational is to trigger a reload of ssl store file by the ZK notification. @cmccabe @rajinisivaram came out this idea to augment the path to
```
//path//to//ssl//store//file
```
when a reload is requested on the receiver broker, and by propagating such a path other brokers would see a difference and thus reload their corresponding store files as well. In the meantime, we need to trim the path back to single slash after handling the notification:
```
/path/to/ssl/store/file
```